### ISSUE TITLE
React v16.0.0 error in PropTypes

### ISSUE THREAD

**Issue by tharyckgusmao** [s-yadav/react-number-format#90]
After upgrading to the react v16.0.0 I got the error return when using the lib:

"uncaught TypeError: Cannot read property 'onOfType' of undefined"

If someone has the error I correct as follows,
fix getting and importing the lib prop-types:

```
var PropTypes = require ('prop-types');
```

and change:

```
var propTypes = {
millimeter: PropTypes.oneOfType ([PropTypes.string, PropTypes.bool]),
decimalSeparator: PropTypes.oneOfType ([PropTypes.string, PropTypes.bool]),
decimalPrecision: PropTypes.oneOfType ([PropTypes.number, PropTypes.bool]),
displayType: PropTypes.oneOf (['input', 'text'])
prefix: PropTypes.string,
suffix: PropTypes.string,
format: PropTypes.oneOfType ([PropTypes.string, PropTypes.func]),
mask: PropTypes.string,
value: PropTypes.oneOfType ([PropTypes.number, PropTypes.string]),
CustomInput: PropTypes.func
};
```

**Comment by s-yadav** [comment]
Will be fixed in v3.0.0

**Comment by s-yadav** [comment]
Fixed
